### DISCUSSION TITLE
Reduce onxyia lockout

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3481] [Suggestions]
Onxyia is no longer current content as of the release of BWL, reduce the lockout so that we can farm those last few specific loot pieces & scales. 

Down to 5 days like was an option on the vote, or even 3 days like ZG is suggested will be.

**Comment by balleny** [comment]
Onyxia drops T2 helmet. Currently progression is T2. Onyxia is the current tier.
